code/common/kafka.q

```diff
@@ -0,0 +1,52 @@
+\d .kafka
+
+// configuration
+enabled:@[value;`enabled;.z.o in `l32`l64]		// whether kafka is enabled
+
```

Comment:
we should have a default definition here of kupd e.g. 

kupd:@[value;`kupd;{{[k;x] -1 `char$x;}}]

to allow this function to be set externally